libs/common/src/sql-db-entities/payment/payment.entity.ts

```diff
@@ -13,19 +13,19 @@ export class PaymentEntity extends AuditableBaseEntity {
   /**
    * The currency the payment was made in
    */
-  @Column({ type: 'nvarchar' })
+  @Column({ nullable: false, type: 'nvarchar' })
```

Comment:
jsut to double check with your changes to the migration, if you run all migrations from a fresh db we don't have any pending migration changes now?